09-Photutils/photutils_extended.ipynb

```diff
@@ -1174,7 +1178,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.5"
+   "version": "3.6.9"
```

Comment:
Hmmm... The Python version should have been 3.7 if you used the environment in the setup instructions. :thinking:

Reply:
hmm thats weird, i did set up the environment but maybe I wasn't in it. I'll figure out what happened.

Reply:
fixed it. the issue was that 'source activate astropy-workshop' claimed to put me in the right environment but python and some packages were out of date. 'conda activate' worked.